### ISSUE TITLE
Application build failed

### ISSUE THREAD

**Issue by elissa2333** [casbin/gorm-adapter#44]
version: 
github.com/casbin/gorm-adapter/v3 v3.0.1

log ：
```# github.com/mattn/go-sqlite3
exec: "gcc": executable file not found in %PATH%
```

The build fails under Windows because I have not installed gcc and cannot build qlite3

Rely on explanation `go mod why github.com/mattn/go-sqlite3`

```
# github.com/mattn/go-sqlite3
myApp/internal/cas
github.com/casbin/gorm-adapter/v3
gorm.io/driver/sqlite
github.com/mattn/go-sqlite3
```

**Comment by hsluoyz** [comment]
It seems that this PR: https://github.com/casbin/gorm-adapter/commit/57a0c400a4d943796ea6787124fe5540517545bc has caused this issue. @L1ghtman2k can you fix it?

**Comment by L1ghtman2k** [comment]
@elissa2333 Have you started getting the above error only recently? In my setup I always had gcc on windows, so I have never encountered that issue.

According to https://github.com/mattn/go-sqlite3#installation, the package requires gcc to be installed, since `go-sqlite3 is cgo package`

You can find the solution for windows here:
According to https://github.com/mattn/go-sqlite3/issues/467

@hsluoyz, I am not sure if this could be labeled as bug, since that is a requirement of a library gorm depends on(Unless there are some workarounds around cgo libraries that doesn't require gcc).

**Comment by hsluoyz** [comment]
@L1ghtman2k can we remove the dependency on: https://github.com/mattn/go-sqlite3 ? I know this adapter supports sqlite, but let's not add it to the code directly. Or comment it, people who need it can uncomment to enable the feature.

This is classified as a bug because Casbin is supposed to work on Windows, Linux, macOS without special setup like gcc or cgo. Looking forward to the PR!

**Comment by L1ghtman2k** [comment]
Oh, got it. My impression was that @elissa2333 was trying to utilize sqlite library, let me get the PR

**Comment by elissa2333** [comment]
> @ elissa2333您最近才开始遇到上述错误吗？在我的设置中，我总是在Windows上使用gcc，所以我从未遇到过该问题。根据https://github.com/mattn/go-sqlite3#installation，该软件包需要安装gcc，因为`go-sqlite3 is cgo package`
> 
> 您可以在此处找到Windows解决方案：
> 根据[mattn / go-sqlite3＃467](https://github.com/mattn/go-sqlite3/issues/467)
> 
> @hsluoyz，我不确定是否可以将其标记为bug，因为这是库gorm依赖的要求（除非围绕cgo库的某些变通办法不需要gcc）。

Thank you for your answer. I encountered an error while upgrading v3. My program only supports MySQL postgres

**Comment by elissa2333** [comment]
> @elissa2333 Have you started getting the above error only recently? In my setup I always had gcc on windows, so I have never encountered that issue. According to https://github.com/mattn/go-sqlite3#installation, the package requires gcc to be installed, since `go-sqlite3 is cgo package`
> 
> You can find the solution for windows here:
> According to [mattn/go-sqlite3#467](https://github.com/mattn/go-sqlite3/issues/467)
> 
> @hsluoyz, I am not sure if this could be labeled as bug, since that is a requirement of a library gorm depends on(Unless there are some workarounds around cgo libraries that doesn't require gcc).

Thank you for your answer. Encountered an error while upgrading v3. My program only supports MySQL Postgres

**Comment by hsluoyz** [comment]
@elissa2333 fixed by commit: https://github.com/casbin/gorm-adapter/commit/007a8c0e7ae1746f1abd21f01bc7c18e36c205b7

Released v3.0.2: https://github.com/casbin/gorm-adapter/releases/tag/v3.0.2
